packages/react-reconciler/src/ReactFiberOffscreenComponent.js

```diff
@@ -31,3 +32,7 @@ export type OffscreenState = {|
   baseLanes: Lanes,
   cachePool: SpawnedCachePool | null,
 |};
+
+export type OffscreenInstance = {|
+  id: number,
```

Comment:
Can you leave this as an empty object for now? Not convinced yet we need to have a numeric id, since we can use the object itself as a key. But even if we do, we can figure that out in the next PR.